app/views/partys/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_with url: "/users/#{@user.id}/movies/#{@movie.id}/viewing-party/new?runtime=#{@movie.runtime}", method: :post, local: true do |form| %>
+  <%= form.label :duration %>
+  <%= form.text_field :duration,placeholder: "#{@movie.runtime}"%>
```

Comment:
This must be the solution to the issue you were having with the instance variable in the form.  Nice work!